lib/kafo/kafo_configure.rb

```diff
@@ -155,9 +155,44 @@ def initialize(*args)
 
       self.class.hooking.execute(:init)
       set_parameters # here the params gets parsed and we need app config populated
+      migrate_answers_to_user_params_file
       set_options
     end
 
+    def migrate_answers_to_user_params_file
```

Comment:
This is my approach to how to migrate an answers file to this new approach. Right now this is "automatic" if an answers file is found and no user params file. This could instead also be triggered by setting a version in the configuration file or some other flag.